### PR TITLE
change charge type to destination charge

### PR DESCRIPTION
**Pull request by btkostner**
Fixes issues causing #483. Avoids `on_behalf_of` due to manual transfers and a bunch of other work.

### CODE REVIEW

**Comment by danrabbit** [comment]
Docs here: https://stripe.com/docs/connect/destination-charges

Sounds right to me logistically :+1:  I'll let @lewisgoddard approve the code

File: src/service/stripe.js [hunk 1]
```diff
@@ -128,14 +128,16 @@ export function postCharge (account: string, token: string, amount: number, curr
...
   .type('form')
   .send({
     amount: cut['total'],
-    application_fee: cut['elementary'],
     currency,
     description,
-    source: token
+    source: token,
+    destination: {
+      account,
```
Comment (lewisgoddard) [inline src/service/stripe.js]: Should this not be `account: account` ?

Comment (lewisgoddard) [inline src/service/stripe.js]: I'm confused as to why this and currency don't seem to have a key.

Comment (btkostner) [inline src/service/stripe.js]: It's an es6 feature. If the key is the same as a variable name you can exclude it. It's pretty much the same as `account: account`

Comment (btkostner) [inline src/service/stripe.js]: http://eslint.org/docs/rules/object-shorthand

Comment (lewisgoddard) [inline src/service/stripe.js]: TIL

**Pull request merged**